src/index.js

```diff
@@ -19,6 +20,13 @@ const client = new ApolloClient({
   link
 })
 
+cache.writeData({
+  data: {
+    currentuser: !!localStorage.getItem("token"),
+    
+  }
+})
```

Comment:
What's this for?

Reply:
I'm Sorry, i was experimenting with local storage on the wrong branch and forgot to change it before pushing